src/resolvers.ts

```diff
@@ -128,7 +190,9 @@ const resolvers = {
 
                 return updatedSubmission
             } catch (error) {
-                throw new Error(`Failed to update submission: ${error}`)
+                console.log(error)
+                console.log('INPUT_FIELDS:', JSON.stringify(args))
+                return CustomErrors.missingInput('Failed to update the submission.')
```

Comment:
Husky doesn't allow me to put a longer string for customerErrors so I had to keep it short

Reply:
we can update eslint in a future ticket to allow longer strings. Personally I think strings can be really long if need be. It's not hard to scroll if we have to :)